qiskit/schemas/backend_configuration_schema.json

```diff
@@ -255,7 +255,26 @@
                                 "type": "array",
                                 "minItems": 0,
                                 "description": "A list of available parametric pulse shapes",
-                                "items": {"type": "string"}}
+                                "items": {"type": "string"}},
+                            "channels": {
```

Comment:
Can you show an example? Is `channels` the most descriptive name for this? It seems really general, and it also prevents us from using `channels` for something else in the future, right?

Reply:
Example:
```
{'d0': {'type': 'drive',
        'purpose': 'drive',
        'operates': {'qubits': [0]}},
 'm0': {'type': 'measure',
        'purpose': 'measure',
        'operates': {'qubits': [0]}},
 'u0': {'type': 'control',
        'purpose': 'cross-resonance',
        'operates': {'qubits': [0, 1]}},
 'acquire0': {'type': 'acquire',
              'purpose': 'acquire',
              'operates': {'qubits': [0]}},
```
Initially the name was `channel_config` but renamed it to `channels` so that its consistent with `gates`. [(more info)](https://github.com/Qiskit/qiskit-terra/pull/3904#discussion_r398065000)

Reply:
Thank you!